pkgs/development/python-modules/qscintilla-qt5/default.nix

```diff
@@ -0,0 +1,34 @@
+{ pkgs, buildPythonPackage, qscintillaCpp,
+  lndir, sip, python, pyqt5 }:
+buildPythonPackage rec {
+  name = "qscintilla-${version}";
+  version = qscintillaCpp.version;
+  src = qscintillaCpp.src;
+  format = "other";
+
+  buildInputs = [ lndir qscintillaCpp sip ];
+  propagatedBuildInputs = [ pyqt5 ];
+
+  preConfigure = ''
+    mkdir -p $out
+    lndir ${pyqt5} $out
+    rm -rf "$out/nix-support"
+    cd Python
+    ${python.executable} ./configure.py \
+      --pyqt=PyQt5 \
+      --destdir=$out/lib/${python.libPrefix}/site-packages/PyQt5 \
+      --stubsdir=$out/lib/${python.libPrefix}/site-packages/PyQt5 \
+      --apidir=$out/api/${python.libPrefix} \
+      --qsci-incdir=${qscintillaCpp}/include \
+      --qsci-libdir=${qscintillaCpp}/lib \
+      --pyqt-sipdir=${pyqt5}/share/sip/PyQt5 \
+      --qsci-sipdir=$out/share/sip/PyQt5
+  '';
+
+  meta = with pkgs.lib; {
+    description = "A Python binding to QScintilla, Qt based text editing control";
+    license = licenses.lgpl21Plus;
+    maintainers = with maintainers; [ lsix ];
+    platforms = platforms.unix;
```

Comment:
Is there a reason for specifying this? `buildPythonPackage` will set an appropriate default.

Reply:
I re-used what was already present in `pkgs/development/python-modules/qscintilla/default.nix`. I can leave it unspecified.